backend/src/main/java/com/hyooun/wilson/review/adapter/out/persistence/ReviewPersistenceAdapter.java

```diff
@@ -0,0 +1,27 @@
+package com.hyooun.wilson.review.adapter.out.persistence;
+
+import com.hyooun.wilson.common.PersistenceAdapter;
+import com.hyooun.wilson.project.domain.Reviewer;
+import com.hyooun.wilson.review.application.port.out.GenerateReviewPort;
+import com.hyooun.wilson.review.domain.ReviewResult;
+import lombok.AllArgsConstructor;
+
+import java.util.List;
+
+@AllArgsConstructor
+@PersistenceAdapter
+class ReviewPersistenceAdapter implements GenerateReviewPort {
```

Comment:
ReviewPersistenceAdapter 클래스에 GenerateReviewPort를 구현하는 generateReview 메서드를 완성하세요.

```suggestion
public ReviewResult generateReview(String title, String body, Reviewer reviewer, List<String> sourceCodes, List<String> references) { return null; }
```